.github/workflows/internal-ci-checks.yml

```diff
@@ -0,0 +1,28 @@
+name: Internal CI Checks
+# This workflows checks if the author of a PR is a member of the CodeQL team
+# and adds `ready-for-internal-ci` label to trigger the internal CI
+
+on:
+  pull_request_target:
+
```

Comment:
I recommend adding a permissions block to request only the permission that are needed for label posting.

Reply:
Thank you! Done: https://github.com/github/codeql/pull/10969/commits/e557e518f4f315ddcbc116967a2596e103238f33